packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.m

```diff
@@ -23,3 +23,132 @@
 
   return request;
 }
+
+extern NSHTTPCookie *_Nullable FWFNSHTTPCookieFromCookieData(FWFNSHttpCookieData *data) {
+  NSMutableDictionary<NSHTTPCookiePropertyKey, id> *properties = [NSMutableDictionary dictionary];
+  for (int i = 0; i < data.propertyKeys.count; i++) {
+    NSHTTPCookiePropertyKey cookieKey =
+        FWFNSHTTPCookiePropertyKeyFromEnumData(data.propertyKeys[i]);
+    if (!cookieKey) {
+      return nil;
+    }
+    [properties setObject:data.propertyValues[i] forKey:cookieKey];
+  }
+  return [NSHTTPCookie cookieWithProperties:properties];
+}
+
+NSKeyValueObservingOptions FWFNSKeyValueObservingOptionsFromEnumData(
+    FWFNSKeyValueObservingOptionsEnumData *data) {
+  switch (data.value) {
+    case FWFNSKeyValueObservingOptionsEnumNewValue:
+      return NSKeyValueObservingOptionNew;
+    case FWFNSKeyValueObservingOptionsEnumOldValue:
+      return NSKeyValueObservingOptionOld;
+    case FWFNSKeyValueObservingOptionsEnumInitialValue:
+      return NSKeyValueObservingOptionInitial;
+    case FWFNSKeyValueObservingOptionsEnumPriorNotification:
+      return NSKeyValueObservingOptionPrior;
+  }
+
+  return -1;
+}
+
+NSHTTPCookiePropertyKey _Nullable FWFNSHTTPCookiePropertyKeyFromEnumData(
+    FWFNSHttpCookiePropertyKeyEnumData *data) {
+  switch (data.value) {
+    case FWFNSHttpCookiePropertyKeyEnumComment:
+      return NSHTTPCookieComment;
+    case FWFNSHttpCookiePropertyKeyEnumCommentUrl:
+      return NSHTTPCookieCommentURL;
+    case FWFNSHttpCookiePropertyKeyEnumDiscard:
+      return NSHTTPCookieDiscard;
+    case FWFNSHttpCookiePropertyKeyEnumDomain:
+      return NSHTTPCookieDomain;
+    case FWFNSHttpCookiePropertyKeyEnumExpires:
+      return NSHTTPCookieExpires;
+    case FWFNSHttpCookiePropertyKeyEnumMaximumAge:
+      return NSHTTPCookieMaximumAge;
+    case FWFNSHttpCookiePropertyKeyEnumName:
+      return NSHTTPCookieName;
+    case FWFNSHttpCookiePropertyKeyEnumOriginUrl:
+      return NSHTTPCookieOriginURL;
+    case FWFNSHttpCookiePropertyKeyEnumPath:
+      return NSHTTPCookiePath;
+    case FWFNSHttpCookiePropertyKeyEnumPort:
+      return NSHTTPCookiePort;
+    case FWFNSHttpCookiePropertyKeyEnumSameSitePolicy:
+      if (@available(iOS 13.0, *)) {
+        return NSHTTPCookieSameSitePolicy;
+      } else {
+        NSLog(@"NSHTTPCookieSameSitePolicy is only supported on iOS 13+.");
```

Comment:
We probably don't want to log about this; people aren't going to want to conditionalize their Dart cookie construction on iOS version. I vote we just silently drop it on unsupported versions.

Reply:
Agreed. I updated the enum with documentation about the version requirement and that it is ignored if used.